save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ManageUserRoleCard.kt

```diff
@@ -177,7 +177,7 @@ fun manageUserRoleCardComponent(
                 it.decodeFromJsonString<String>()
             }
             .toRole()
-        setSelfRole(role)
+        setSelfRole(listOf(role, props.selfUserInfo.globalRole ?: Role.NONE).maxByOrNull { it.priority }!!)
```

Comment:
And this, imo, is super unobvious, see the previous comment

Reply:
Yes, ok, makes sense